pkg/types/hashedrekord/v0.0.1/entry_test.go

```diff
@@ -93,7 +93,7 @@ func TestCrossFieldValidation(t *testing.T) {
 	h := sha256.Sum256(dataBytes)
 	dataSHA := hex.EncodeToString(h[:])
 
-	signer, _ := signature.LoadSigner(key, crypto.SHA256)
+	signer, _ := signature.LoadSigner(key, crypto.SHA256, signature.LoadDefaultSV, nil)
```

Comment:
Probably we can avoid all these changes if we add a `LoadSignerWithOpts`/`LoadVerifierWithOpts`/`LoadSignerVerifierWithOpts` instead of changing the existing ones.